go/adbc/driver/bigquery/driver.go

```diff
@@ -0,0 +1,222 @@
+	case "true":
+		return true, nil
+	case "false":
+		return false, nil
+	default:
+		return false, adbc.Error{
+			Code: adbc.StatusInvalidArgument,
+			Msg:  fmt.Sprintf("Invalid boolean value, expected `true` or `false`, got `%s`", value),
+		}
+	}
+}
+
+func stringToTable(value string) (*bigquery.Table, error) {
+	parts := strings.Split(value, ".")
+	if len(parts) != 3 {
+		return nil, adbc.Error{
+			Code: adbc.StatusInvalidArgument,
+			Msg:  fmt.Sprintf("Invalid Table Reference format, expected `ProjectId.DatasetId.TableId`, got: `%s`", value),
+		}
+	}
+
+	return &bigquery.Table{
+		ProjectID: parts[0],
+		DatasetID: parts[1],
+		TableID:   parts[2],
+	}, nil
+}
+
+func stringToTableCreateDisposition(value string) (bigquery.TableCreateDisposition, error) {
+	switch value {
+	case "CREATE_NEVER":
+		return bigquery.CreateNever, nil
+	case "CREATE_IF_NEEDED":
+		return bigquery.CreateIfNeeded, nil
+	default:
+		return bigquery.CreateIfNeeded, adbc.Error{
+			Code: adbc.StatusInvalidArgument,
+			Msg:  fmt.Sprintf("unknown table create disposition value `%s`", value),
+		}
+	}
+}
+
+func stringToTableWriteDisposition(value string) (bigquery.TableWriteDisposition, error) {
+	switch value {
+	case "WRITE_APPEND":
+		return bigquery.WriteAppend, nil
+	case "WRITE_TRUNCATE":
+		return bigquery.WriteTruncate, nil
+	case "WRITE_EMPTY":
+		return bigquery.WriteEmpty, nil
+	default:
+		return bigquery.WriteEmpty, adbc.Error{
+			Code: adbc.StatusInvalidArgument,
+			Msg:  fmt.Sprintf("unknown table write disposition value `%s`", value),
+		}
+	}
+}
+
+func stringToQueryPriority(value string) (bigquery.QueryPriority, error) {
+	switch value {
+	case "BATCH":
+		return bigquery.BatchPriority, nil
+	case "INTERACTIVE":
+		return bigquery.InteractivePriority, nil
+	default:
+		return bigquery.InteractivePriority, adbc.Error{
+			Code: adbc.StatusInvalidArgument,
+			Msg:  fmt.Sprintf("unknown table create disposition value `%s`", value),
+		}
+	}
+}
+
+func boolToString(value bool) string {
+	if value {
+		return "true"
+	}
+	return "false"
+}
+
+func tableToString(value *bigquery.Table) string {
+	if value == nil {
+		return ""
+	} else {
+		return fmt.Sprintf("%s.%s.%s", value.ProjectID, value.DatasetID, value.TableID)
+	}
+}
+
+func tableCreateDispositionToString(value bigquery.TableCreateDisposition) string {
+	switch value {
+	case bigquery.CreateNever:
+		return "CREATE_NEVER"
+	case bigquery.CreateIfNeeded:
+		return "CREATE_IF_NEEDED"
+	}
+	return ""
+}
+
+func tableWriteDispositionToString(value bigquery.TableWriteDisposition) string {
+	switch value {
+	case bigquery.WriteAppend:
+		return "WRITE_APPEND"
+	case bigquery.WriteTruncate:
+		return "WRITE_TRUNCATE"
+	case bigquery.WriteEmpty:
+		return "WRITE_EMPTY"
+	}
+	return ""
+}
+
+func queryPriorityToString(value bigquery.QueryPriority) string {
+	switch value {
+	case bigquery.BatchPriority:
+		return "BATCH"
+	case bigquery.InteractivePriority:
+		return "INTERACTIVE"
+	}
+	return ""
+}
```

Comment:
same comment, these functions are unnecessary and can be replaced entirely with just casting the value to `string`